QUEST.tsv

```diff
@@ -122,82 +122,82 @@ xQUEST_20150317_000121	This mission makes me feel like I've seen enough monsters
 xQUEST_20150317_000122	Chupacabras are fond of lush vegetation and upper streams.
 xQUEST_20150317_000123	Klaipeda Guard Captain
 xQUEST_20150317_000124	You're the one who…
-xQUEST_20150317_000125	It looks like you can see into the upper forest from here. {nl}Try not to provoke anything bigger than you.
-xQUEST_20150317_000126	You don't have the required quest item. Are you sure you want to continue?
+xQUEST_20150317_000125	It looks like you can see into the upper forest from here. {nl}Try not to get into trouble.
+xQUEST_20150317_000126	You don't have the required quest item. Are you sure you want to go on?
 xQUEST_20150317_000127	Klaipeda Girl
 xQUEST_20150317_000128	The man who lives next door to me used to be a monster hunter. {nl}He's retired now, though. It was too dangerous.
 xQUEST_20150317_000129	I think it would be nice if all these Revelators were a sign that the goddesses are back.
-xQUEST_20150317_000130	It might be silly to rely on hope, but it's better than being hopeless.
+xQUEST_20150317_000130	It might be silly to hope for the Goddess again, but it's better than being hopeless.
 xQUEST_20150317_000131	Klaipeda Lady
 xQUEST_20150317_000132	We can still grow from the land, and have a warm hearth. {nl}I will always believe in the Goddess. 
 xQUEST_20150317_000133	Keep faith. {nl}The Goddess may still send a savior to us.
 xQUEST_20150317_000134	Hoplite Master
-xQUEST_20150317_000135	If the sword is a symbol of power, then the spear is of pure destruction. {nl}Power or destruction. What would you reach for, if you were on the battlefield?
+xQUEST_20150317_000135	If the sword is a symbol of power, then the spear is pure destruction. {nl}Power and destruction. What would you reach for, if you were on the battlefield?
```

Comment:
Old version is better -> it gives the players a choice between sword and spear. Also spears are not pure destruction but a symbol of pure destruction, counterpart to symbol of power for sword.